server/game/cards/04.2-TL/ApprenticeEarthcaller.js

```diff
@@ -1,10 +1,25 @@
 const DrawCard = require('../../drawcard.js');
 
 class ApprenticeEarthcaller extends DrawCard {
-    setupCardAbilities(ability) { // eslint-disable-line no-unused-vars
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'Set skill values to printed values',
+            condition: () => this.game.isDuringConflict(),
+            target: {
+                cardType: 'character',
+                cardCondition: card => card.isAttacking() && card.attachments.size() === 0,
+                gameAction: ability.actions.cardLastingEffect(context => ({
+                    effect: [
+                        ability.effects.setMilitarySkill(context.target.printedMilitarySkill()),
```

Comment:
`printedMilitarySkill` is a variable not a function, so it should just be `(context.target.printedMilitarySkill)`